src/HTML5Backend.js

```diff
@@ -34,25 +34,26 @@ export default class HTML5Backend {
     this.endDragNativeItem = this.endDragNativeItem.bind(this);
   }
 
-  setup() {
-    if (typeof window === 'undefined') {
+  setup(target = window) {
```

Comment:
@Vadorequest I see this is closed, but I'm doing somewhere where this would be really useful. I'm just wondering how you would wire things up to pass in an alternative target?